app/controllers/passwordless/sessions_controller.rb

```diff
@@ -96,5 +96,10 @@ def find_session
         token: params[:token]
       )
     end
+
+    def session_expired
+      flash[:error] = 'Your session has expired, please sign in again.'
```

Comment:
I think we found our next issue: Adding `I18n` locale support 😄

Reply:
🤣